src/bjointsp/parameters/templates/fw1chain.yaml

```diff
@@ -9,22 +9,31 @@ vnfs:
     inputs_bwd: 0
     outputs_fwd: 1
     outputs_bwd: 0
-    cpu: [0]
+    cpu: [1]
     mem: [0]
+    vnf_delay: 50
```

Comment:
Similarly, I don't think setting "vnf_delay" for the user "vnf" makes sense.

 In fact, maybe you could add a check in the `reader.py` that checks if all resource requirements and vnf_processing delays for user/source vnfs are set to 0. If this is not the case, log a warning message (but don't throw an exception)